yarn.lock

```diff
@@ -8346,58 +8346,6 @@ doctrine@^3.0.0:
   dependencies:
     esutils "^2.0.2"
 
-docusaurus@^2.0.0-alpha.61:
```

Comment:
hi,

why is this needed in this PR?

Reply:
will be fixed in https://github.com/facebook/docusaurus/pull/3197

Reply:
Ah yep, I'll remove